github/repos_releases.go

```diff
@@ -46,6 +47,19 @@ func (r RepositoryRelease) String() string {
 	return Stringify(r)
 }
 
+// RepositoryReleaseNotes represents a GitHub-generated release notes.
+type RepositoryReleaseNotes struct {
+	Name string `json:"name,omitempty"`
```

Comment:
If these fields are optional, which is usually the case when `omitempty` is involved, then they need to be pointers. In other words, they should be `*string` and not `string`... for all 5 new parameters.

```suggestion
	Name *string `json:"name,omitempty"`
```

Reply:
this seems to be required